scripts/workflow_config.json

```diff
@@ -3,11 +3,10 @@
   "emod3d_version" : "3.0.4",
   "global_root" : "/nesi/project/nesi00213",
   "bin_process_path" : "/nesi/project/nesi00213/opt/maui/emod3d/3.0.4-gcc/bin/",
-  "qcore_lib_path" : "/nesi/project/nesi00213/opt/maui/qcore",
+  "qcore_lib_path" : "/home/melody.zhu/qcore",
```

Comment:
Have to reupdate this again. Ideally, let's find a way to stop this being committed in the future.